### DISCUSSION TITLE
Why are has_many default keys reversed from Orator?

### DISCUSSION THREAD

**Discussion by willjohnathan** [MasoniteFramework/orm#815] [General]
Sorry, I have scanned through the documentation and issues and haven't seen this discussed. Seems the current convention will always require the keys to be explicitly defined. Also, this is opposite of the Orator documentation [here](https://orator-orm.com/docs/0.9/orm.html#one-to-many):

```
@has_many('foreign_key')
def comments(self):
    return Comment

@has_many('foreign_key', 'local_key')
def comments(self):
    return Comment
```

and not mentioned as a breaking change in the Masonite-ORM documentation. 

Where as Masonite-ORM requires the following:

```
class Client(Model):
    
    @has_many('id', 'client_id')
    def projects(self):
        from src.models.Project import Project
        return Project
```

**Comment by josephmancuso** [comment]
Masonite ORM is foreign_key, local_key for the has many relationship

**Comment by willjohnathan** [comment]
Hmm, okay, I guess there is something subtle I am missing here (version: masonite-orm==2.18.6). 

A dumbed down script outputs:
```
from masoniteorm.models import Model
from masoniteorm.relationships import has_many

class Project(Model):
    pass

class Client(Model):
    @has_many('id', 'client_id')
    def projects_flipped(self):
        return Project

    @has_many
    def projects(self):
        return Project

client = Client.find(3)
client.related('projects_flipped').to_sql() #== SELECT * FROM `projects` WHERE `projects`.`client_id` = '3'
client.related('projects').to_sql() #== SELECT * FROM `projects` WHERE `projects`.`projects_id` = '3' ->expecting `projects`.`client_id` = 3
```
